src/xmipp/applications/programs/cuda_volume_halves_restoration/cuda_volume_halves_restoration.cpp

```diff
@@ -0,0 +1,28 @@
+/***************************************************************************
```

Comment:
this file has to have _\_main_ suffix.
If not, it will not be picked up by the build system. That's why you got 0 issues in Sonarcloud